### ISSUE TITLE
deeplab doesn't predict correctly the segmentation masks

### ISSUE THREAD

**Issue by essalahsouad** [tensorflow/models#6848]
### **System information**
**What is the top-level directory of the model you are using:**
https://github.com/tensorflow/models/tree/master/research/deeplab
**Have I written custom code (as opposed to using a stock example script provided in TensorFlow):**
No
**OS Platform and Distribution (e.g., Linux Ubuntu 16.04):**ubuntu 18.04
**TensorFlow installed from (source or binary):**binary
**TensorFlow version (use command below):**1.13.1 CPU
**Bazel version (if compiling from source):**N/A
**CUDA/cuDNN version:**N/A
**GPU model and memory:**N/A

**### Exact command to reproduce:**
run models/research/deeplab/deeplab_demo.ipynb

**### Describe the problem**
I tried train DeeplabV3 pre-trained model on a new class(eyes iris)
my dtat=RGB iris image+gray-scale iris label:1500 images
as a result of the training step, a got those files:
graph.pbtxt
model.ckpt-1000.data-00000-of-00001
model.ckpt-1000.info
model.ckpt-1000.meta
that i converted to frozen_inference_graph-1000.pb
in the test step i got this:
![Capture du 2019-05-22 08-39-51](https://user-images.githubusercontent.com/34586193/58156169-69cf8d00-7c6d-11e9-86b4-e175baeac0fc.png)
it's a good result for me but what I need to segment is something like this:
i need to segment only the iris 
![Capture du 2019-05-22 08-43-44](https://user-images.githubusercontent.com/34586193/58156316-bca94480-7c6d-11e9-8346-b5d48a12f724.png)
(this screenshot it's from my training mask)
